Dockerfile

```diff
@@ -1,10 +1,22 @@
-FROM alpine:3.5
+FROM golang:1.12-alpine as builder
 
-MAINTAINER wata727
+RUN apk --no-cache add git make gcc musl-dev zip
+
+WORKDIR /go/src/github.com/wata727/tflint/
+
+ADD . /go/src/github.com/wata727/tflint
+
+RUN make release
```

Comment:
`make release` builds binaries for all architectures. It might be better to just run `make build` for the linux platform.